aup.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Acceptable Use Policy"
+---
+
+The terms "R-multiverse", "User", "Contributor", "Moderator" and "Administrator" shall have the meaning as defined in the [R-multiverse Governance Document](governance.md).
+This Acceptable Use Policy (the "Policy") outlines the standards of conduct expected from all Users and Contributors to R-multiverse (the "Repository").
```

Comment:
This document deals with "conduct" required of Users and Contributors, not everyone like the CoC. It may sound the same, but again, I don't want to invent another term, and defining it too narrowly also creates risks.